ports/matplotplusplus/vcpkg.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "matplotplusplus",
+  "version-string": "2021-03-07",
```

```suggestion
  "version-date": "2021-03-07",
```
This should be version-date in this case. Please see https://github.com/microsoft/vcpkg/blob/master/docs/specifications/versioning.md#22-package-versions